en/sensor/airspeed.md

```diff
@@ -70,6 +72,8 @@ The specific configuration for sensors that do not have a separate page is liste
 
 Airspeed sensors should be calibrated by following the instructions: [Basic Configuration > Airspeed](../config/airspeed.md).
 
-## Developer Information
+## See Also
 
+- [Using the ECL EKF > Airspeed](../advanced_config/tuning_the_ecl_ekf.md#airspeed)
 - [Airspeed drivers](https://github.com/PX4/PX4-Autopilot/tree/main/src/drivers/differential_pressure) (source code)
+- [VTOL Without an Airspeed Sensor](../config_vtol/vtol_without_airspeed_sensor.md)
```

Comment:
While we shouldn't provide detail in this page, I think it makes sense to link to related pages.